server/pyproject.toml

```diff
@@ -60,5 +60,8 @@ dev = {composite = ["db", "wait", "migrate", "start"], env = {DB_HOST = "localho
 test = {cmd = "coverage run manage.py test -v 2", env = {DB_HOST = "localhost"}}
 citest = {cmd = "python manage.py test -v 2 --keepdb"}
 post_test = {cmd = "coverage report --skip-empty --omit=tests/*,*tests*"} #choose additional files to skip here with "," and no spaces)
+linter = {cmd = "ruff check ."}
 
+[tool.ruff]
+line-length = 119 # Alternatives PEP-8: 79, Black/Ruff: 88
```

Comment:
119 is the recommended length for Django. It is also the max char width for GitHub code review without needing to horizontal scroll. The alternatives will split many lines of Django in the server.